### DISCUSSION TITLE
This has serious potential, and I hope you get more funding/support.

### DISCUSSION THREAD

**Discussion by dogman035** [blackears/cyclopsLevelBuilder#94] [General]
So, I've been following [Chris Bryant's](https://twitter.com/LiterallyCrypt) stuff on twitter recently. And it's made me realize how lacking level design tools are in gamedev.

There is very little in the way of good options.

Trenchbroom works for simpler square levels, but not anything with curved geometry.

Blender is great at _models,_ but terrible at anything you can't rotate around fully with middle click. So it's bad for level design.

I think what you have so far is great. It's _already_ on par with Trenchbroom, and maybe a bit better even at some things. So I really hope that, eventually, this becomes something like Godot's Hammer. Because god damn does the world need it.

**Comment by blackears** [comment]
Thanks. I started writing it because I didn't like the existing options too.  I do have some ideas for going beyond just convex blocks, but want to make sure the core is solid before going beyond that.

**Reply by codecat** [reply]
As the creator of [a TrenchBroom loader addon](https://github.com/codecat/godot-tbloader), I agree that there aren't many good tools for this. (Even my TBLoader feels like a weird hack around something that should be built-in.)

I'd happily support you financially on Patreon to see this project grow, if you still intend to keep working on it! (If you made a higher Patreon tier than $1 😊)

**Reply by blackears** [reply]
Thanks.  Sorry for the late reply - I've been pretty busy lately but think I finally have some time to focus back on the project again.  That's a good idea about Patreon - I should set up a better tier structure.

**Reply by ashelleyPurdue** [reply]
Oh, hey, you're the TBLoader creator!  Thank you so much for your plugin!  It's made my current project feasible(although you're right, it _does_ feel weird and wonky.)

And thank you too, @blackears , for this plugin as well!  This really looks like it's going places!
